### PR TITLE
Release notes 0.71.0

### PR DESCRIPTION
**Pull request by hsaraogi**
**Priority (whenever / two weeks / yesterday)**: today

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/2791)

### CODE REVIEW

File: docs/source/release_notes/release-notes.rst [hunk 1]
```diff
@@ -84,8 +114,7 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2782>`__)
 
     *    - |new|
-         - A CLI to read the punch table was added. The CLI receives an epoch time, in millis, and returns the timestamp that was added to the punch table
-           right before it.
+         - Added a CLI to read the punch table. The CLI receives an epoch time, in millis, and returns the latest AtlasDB timestamp before it.
```
Comment (jeremyk-91) [inline docs/source/release_notes/release-notes.rst]: nit: returns *an approximation of* the latest AtlasDB timestamp...

File: docs/source/release_notes/release-notes.rst [hunk 2]
```diff
@@ -50,10 +50,40 @@ develop
...
+
+    *    - Type
+         - Change
+
+    *    - |new|
+         - **AtlasDB QoS**: AtlasDB now allows clients to live-reloadably configure read and write limits (in terms of bytes) to rate-limit requests to Cassandra.
+           AtlasDB clients will receive a ``RateLimitExceededException`` for requests that are throttled and should handle them appropriately.
+           We provide an exception mapper ``RateLimitExceededExceptionMapper`` to map the throttling exceptions to ``429``, but it is upto the application to register the exception mapper.
+           Note that this is an experimental feature and applications should generally not enable it by default yet, unless the application has hard read-write limits.
+           This should allow us to throttle dynamically in situations where the load on Cassandra is high. In the future, once applications have
```
Comment (jeremyk-91) [inline docs/source/release_notes/release-notes.rst]: Think this sentence is unfinished. I'm guessing you want to mention the idea of a centralized QoS service in the future?

Comment (hsaraogi) [reply docs/source/release_notes/release-notes.rst]: Skipping that, we'll announce when we have it.

File: docs/source/release_notes/release-notes.rst [hunk 3]
```diff
@@ -50,10 +50,40 @@ develop
...
+           Note that this is an experimental feature and applications should generally not enable it by default yet, unless the application has hard read-write limits.
+           This should allow us to throttle dynamically in situations where the load on Cassandra is high.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2629>`__)
+
     *    - |improved|
-         - AtlasDB publish of new releases is now done through the internal circle build. Before it was done via the external circle build.
+         - AtlasDB publish of new releases is now done through the internal circleCI build instead of external circleCI.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2783>`__)
 
+    *    - |improved|
```
Comment (jeremyk-91) [inline docs/source/release_notes/release-notes.rst]: sorry, one last thing: I think this should have the `|logs|` tag

**Pull request merged**